tests/admin_views/tests.py

```diff
@@ -6717,6 +6717,18 @@ def test_redirect_on_add_view_continue_button(self):
         name_input_value = name_input.get_attribute("value")
         self.assertEqual(name_input_value, "Test section 1")
 
+    @screenshot_cases(["desktop_size", "mobile_size", "rtl", "dark"])
+    def test_user_change_password(self):
+        self.admin_login(
+            username="super", password="secret", login_url=reverse("admin:index")
+        )
+
+        change_url = reverse("admin:auth_user_change", args=(self.superuser.pk,))
+        with self.wait_page_loaded():
+            self.selenium.get(self.live_server_url + change_url)
+
+        self.take_screenshot("user_change")
```

Comment:
This test doesn't have any assertions. Instead I'd add `take_screenshot` to one of existing tests, e.g. `UserAdminTest.test_save_button()`.

Reply:
`UserAdminTest` is not a `AdminSeleniumTestCase`, I can't take screenshot in it.